documentation/smg/docs_discussion.md

```diff
@@ -0,0 +1,203 @@
+* Difficult to check how branches would appear
+* Requires manually updating version number in Jenkins
+
+#### Build/Deploy on Horace release
+Tie docs deploy into the existing deploy pipeline such that on each major Horace release docs are released to the right tag
+
+##### Pros:
+* Fully automated
+* Robust against accidental docs damage
+
+##### Cons:
+* Dev docs may grow out of sync in time between major releases
+* Dev docs may be difficult to track
+
+## Docs versioning
+### Current state
+#### Version folder and stable folder
+Currently we have a folder for each version and stable both updated/overwritten on a deploy
+
+##### Pros:
+* Easy to track
+* Pre-release can be set up as a prVersion and cleaned up on release
+
+##### Cons:
+* Manual cleanup of PR
+* Extra effort to make dev changes public
+
+### Alternatives
+#### Unstable, stable, version
+Have an unstable (dev) folder which is updated every push, stable updated on release and version updated every push
+
+##### Pros:
+* Dev changes immediately update through unstable version
+* Stable version for users directs to most recent
+
+##### Cons:
+* More Jenkins logic required
+* Extra folders
+
+#### Dev, Version
+Only have version number and dev version users look up their version
+
+##### Pros:
+* Users of old versions have no confusion on release update
+* Dev changes visible
+
+##### Cons:
+* No short cut to latest version (need to know your version)
+* Discourages users staying up to date?
+
+#### Per-branch docs
+Each branch gets built as branch-docs (mostly relevant if horace-docs merged into main) 
+
+##### Pros:
+* Automatically labelled and searchable
+* All docs build and are reachable
+
+##### Cons:
+* Cleanup of merged/deleted branches
+* Users may not know which branch is relevant
+* Too many docs
+
+#### Per-tag docs
+As above, but with each milestone/tag
+
+##### Pros:
+* As above
+##### Cons:
+* As above
+
+## Stable/Dev status
+### Current state
+#### Full folder copy
+Currently the stable docs are a full copy of the html folder
+
+##### Pros:
+* Hyperlink says stable, not version number
+
+##### Cons:
+* Potential to become out of sync
+* Difficult to redirect stable without full rebuild/deploy
+
+### Alternatives
+#### Redirect
+Just have an HTML redirect file to the lates
+
+##### Pros:
+* Easy to update/fix and can be separated manually from deployment procedure
+* Still easy to automate if desired
+
+###### Cons:
+* Address bar may not show stable on redirect causing confusion
+
+#### Symlink (shortcut)?
+Have a stable folder which links to version without 
+
+##### Pros:
+* All of redirect
+* Address bar correct
+
+###### Cons:
+* Not sure if possible
+
+## Repo access
+### Current state
+#### Open
+Full write access to main
+
+##### Pros:
+* Easy to force an update through
+
+##### Cons:
+* Easy to force an update through
+
+### Alternatives
+#### PR required
+Disable pushing to main
```

Comment:
If the docs are in a separate repo or Horace, we should always be using PRs. If not only for preventing damage to master (as you say), but also for visibility of changes for the rest of the team.